src/main/webapp/index.html

```diff
@@ -71,7 +70,13 @@ <h1>EatGoWhere</h1>
                   <label for="gluten-free">Gluten Free</label>
                 </div>
               </fieldset>
-              <div id="price-filter-container" width="100%"">
+              <fieldset>
+                <legend>Maximum Distance</legend>
+                <div id ="distance-filter-container" width="100%">
+                  <input id="distance-filter" type="number" step="100" min="100" value="500"> meters
+                </div>
+              </fieldset>
+              <div id="price-filter-container" width="100%">
```

Comment:
Have you experimented using sliders to implement this? It will be more intuitive and user-friendly.

Reply:
My concern was that for sliders I have to set a min and max value, and I couldn't think of a good value to set for those two. If I set the range to be too small, it makes the filter redundant - on the other hand, if I set it to be too big, the sliders become too sensitive.

Reply:
What do you think about having the range set to 100m - 1000(or up to 1500)m, with steps of 100?

Any less for the min distance it might make it always generate no results.
Any further for the max distance it'll make more sense for them to just change the origin location instead.

Reply:
Changed distance filter to slider with range 100m-2000m